test/integration/README.md

```diff
@@ -1,8 +1,6 @@
 # Requirements
```

Comment:
Do we really want to have a separate integration test README?

Reply:
I'm going to move the README information into the top level README where it will be more useful. I'll also add a little bit about Jest.

Reply:
I moved it